alot/commands/globals.py

```diff
@@ -40,14 +40,14 @@
 
 @registerCommand(MODE, 'exit')
 class ExitCommand(Command):
-    """Shut down cleanly.
-
-    The _prompt variable is for internal use only, it's used to control
-    prompting to close without sending, and is used by the BufferCloseCommand
-    if settings change after yielding to the UI.
-    """
+    """shut down cleanly"""
 
     def __init__(self, _prompt=True, **kwargs):
+        """
+        :param _prompt: For internal use only, used to control prompting to
+        close without sending, and is used by the BufferCloseCommand if
+        settings change after yielding to the UI.
+        """
```

Comment:
Generally in sphinx documentation the `__init__()` method is document as part of the class documentation, and not separately. It appears that sphinx simply appends the `__init__()` documentation to the class documentation. I don't have a strong opinion either way, as long as we're consistent. @pazz, @lucc opinions?

Reply:
Oh. Missed this, and didn’t know the convention. The change is in any case unnecessary now, because I only moved the parameter description so that it doesn’t leak into the user documentation, but we’ve since agreed to move the user documentation out of the docstring and into the `@registerCommand` decorator.